<launch>
    <!-- pkg: 패키지 이름, type: 실행시킬 파일, name: 노드 이름, output: 터미널창에 띄우기 -->
    <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch"/>
    <node pkg="morai_woowa" type="location.py" name="location_node"></node>
    <node pkg="morai_woowa" type="camera3.py" name="YOLO_node"></node>
    <node pkg="morai_woowa" type="sensor_main" name="sensor_node" output="screen"></node>
</launch>
